### ISSUE TITLE
Linting on Windows broken

### ISSUE THREAD

**Issue by pipobscure** [AtomLinter/linter-jscs#69]
https://github.com/AtomLinter/linter-jscs/blob/master/lib/linter-jscs.coffee#L76 sets the path incorrectly if on windows. This results in silent non-linting.

Why is this line even necessary? The filePath this method gets as an argument is already absolute.

so replaceing this line with

    if !condition then path=undefined

should be sufficient

**Comment by joepaolicelli** [comment]
Linting silently failed for me as well on Windows.

Installing v1.12.0 instead of the current version (v.1.13.0, where the cited line of code was introduced) fixed the problem. ( apm install linter-jscs@1.12.0 )

**Comment by joepaolicelli** [comment]
Actually, even when this line of code is fixed and linting works, custom configuration files are not recognized. I was able to get a custom file working by editing the package code and hardcoding the path to the file in.

I think the issue may be related to the use of `@cwd` (the coffeescript equivalent to `this.cwd`). Its addition in the line cited by @pipobscure broke the entire package on Windows, and it is used to determine the path to search for the configuration files (which worked fine when I replaced `@cwd` with a hardcoded path). It doesn't seem to be used anywhere else - so it seems suspicious that replacing it fixed the only two issues I've had with this package.
